guides/v2.3/graphql/reference/products.md

```diff
@@ -195,6 +197,7 @@ Attribute | Data type | Description
 `special_from_date` | String | The beginning date that a product has a special price
 `special_price` | Float |  The discounted price of the product
 `special_to_date` | String | The end date that a product has a special price
+`stock_status` | ProductStockStatus | An enumeration describing the stock status of the product. Possible values are `IN_STOCK` and `OUT_OF_STOCK`
```

Comment:
Seems weird that `ProductStockStatus` is a "data type", but I trust ya ;)

You should probably add a period at the end of the second clause to match other items, like Price.

Reply:
It's the name of an object. There are other object names in the table, and they're complex enough to have a section in this topic or another to describe them. This one is merely an enumeration value that's either `IN_STOCK` or `OUT_OF_STOCK`.  I don't know why this isn't a simple string, but that's what the code says.